common/djangoapps/util/enterprise_helpers.py

```diff
@@ -0,0 +1,122 @@
+from django.conf import settings
+from django.utils.translation import ugettext as _
+
+try:
+    from enterprise.models import EnterpriseCustomer
+    from enterprise.tpa_pipeline import (
+        active_provider_requests_data_sharing,
+        active_provider_enforces_data_sharing,
+        get_enterprise_customer_for_request,
+    )
+
+except ImportError:
+    pass
+
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+
+
+def enterprise_enabled():
+    """
+    Determines whether the Enterprise app is installed
+    """
+    return 'enterprise' in settings.INSTALLED_APPS
```

Comment:
We need to consistent here in terms of either we should use the feature_flag or check into the installed apps as going here ?  As per discussion with @bradenmacdonald [HERE](https://github.com/edx/edx-platform/pull/14064#discussion-diff-90092342R22)